aviary/segments.py

```diff
@@ -42,7 +42,8 @@ def forward(self, x: Tensor, index: Tensor) -> Tensor:
         return out
 
     def __repr__(self) -> str:
-        return self.__class__.__name__
+        gate_nn, message_nn = self.gate_nn, self.message_nn
```

Comment:
why do we need to assign these variables?

Reply:
Just to avoid a line break in the string. Can get rid of if you prefer.